[Aggregate-reports]Spell check public API failing in nightly runs

Aggregate-reports nightly test runs are failing with:

> Error message:
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1:46 - Unknown word (easm)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:20:18 - Unknown word (Alexa)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:22:5 - Unknown word (alexa)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:54:17 - Unknown word (Whois)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:321:171 - Unknown word (Easm)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:696:5 - Unknown word (alexa)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:696:24 - Unknown word (Alexa)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:700:17 - Unknown word (Whois)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:746:5 - Unknown word (whois)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:748:5 - Unknown word (whois)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:758:13 - Unknown word (Easm)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1091:5 - Unknown word (asns)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1099:5 - Unknown word (cnames)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1135:5 - Unknown word (nxdomain)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1167:5 - Unknown word (alexa)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1257:5 - Unknown word (asns)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1291:5 - Unknown word (nxdomain)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1325:5 - Unknown word (asns)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1331:17 - Unknown word (Whois)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1921:5 - Unknown word (asns)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1933:5 - Unknown word (cnames)
> ##[error].\sdk\easm\defender-easm-rest\review\defender-easm.api.md:1949:10 - Unknown word (Asns)
> ##[error]Spelling errors detected. To correct false positives or learn about spell checking see: https://aka.ms/azsdk/engsys/spellcheck
> ##[error]PowerShell exited with code '1'.

For more details check here:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3171391&view=results

--- a/.vscode/cspell.json
+++ b/.vscode/cspell.json
@@ -436,6 +436,21 @@
       "words": [
         "protobuf"
       ]
+    },
+    {
+      "filename": "sdk/easm/defender-easm-rest/review/defender-easm.api.md",
+      "words": [
+        "Alexa",
+        "alexa",
+        "Asns",
+        "asns",
+        "cnames",
+        "Easm",
+        "easm",
+        "nxdomain",
+        "Whois",
+        "whois"
+      ]
     }
   ]
 }

